steps/zoom/meeting/create/main.go

```diff
@@ -19,92 +22,111 @@ type Args struct {
+		return status.FromError(codes.InvalidArgument, "Environment variables are missing or invalid", err)
 	}
 
 	if z.args.ZoomToken == "" {
-		return nil
+		return status.FromError(codes.Internal, "Zoom token not found", fmt.Errorf("zoom token not found"))
 	}
 
 	if err := json.Unmarshal([]byte(z.args.ZoomToken), &z.token); err != nil {
-		return fmt.Errorf("unmarshal token: %w", err)
+		return status.FromError(codes.Unauthenticated, "Failed to unmarshal zoom token", err)
 	}
 
-	return nil
+	return status.OK
 }
 
-func CreateMeeting(client *http.Client, topic, user string) (*http.Response, error) {
+func CreateMeeting(client *http.Client, topic, user string) (*http.Response, step.Status) {
 	params := zoom.CreateMeetingOptions{
 		Type:  zoom.MeetingTypeInstant,
 		Topic: topic,
 	}
 
 	var buf bytes.Buffer
 	if err := json.NewEncoder(&buf).Encode(params); err != nil {
-		return nil, fmt.Errorf("marshal params: %w", err)
+		return nil, status.FromError(codes.Internal, "Failed to marshal params", err)
 	}
 
 	resp, err := client.Post(fmt.Sprintf(base.ZoomAPIV2+zoom.CreateMeetingPath, user), "application/json", &buf)
 	if err != nil {
-		return nil, fmt.Errorf("post request: %w", err)
+		return nil, status.FromError(codes.Internal, "Failed to post request", err)
 	}
 
-	return resp, nil
+	return resp, status.OK
 }
 
-func (z *ZoomMeetingCreate) Run() (exitCode int, output []byte, err error) {
-	if z.args.ZoomApiKey != "" || z.args.ZoomApiSecret != "" {
-		return z.DeprecatedRun()
-	}
+func (z *ZoomMeetingCreate) Run() (step.Output, step.Status) {
+	stepOutput := &Outputs{}
 
-	if z.args.ZoomToken == "" {
-		return step.ExitCodeFailure, nil, fmt.Errorf("zoom token not found")
+	if z.args.ZoomAPIKey != "" || z.args.ZoomAPISecret != "" {
+		exitCode, output, err := z.DeprecatedRun()
+		if err != nil || exitCode != 0 {
+			return nil, status.FromError(codes.Internal, "Run failed: ", err)
+		}
+		stepOutput.Meeting = output
+		return stepOutput.toMap(), status.OK
 	}
 
 	client := oauth2.NewClient(context.Background(), oauth2.StaticTokenSource(z.token))
 
-	resp, err := CreateMeeting(client, z.args.MeetingTopic, z.args.HostEmail)
-	if err != nil {
-		return step.ExitCodeFailure, nil, fmt.Errorf("create meeting: %w", err)
+	resp, stepStatus := CreateMeeting(client, z.args.MeetingTopic, z.args.HostEmail)
+	if stepStatus.Failed() {
+		return nil, stepStatus
 	}
 
 	respBody, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return step.ExitCodeFailure, nil, fmt.Errorf("read response body: %w", err)
+		return nil, status.FromError(codes.Internal, "Failed to read response body", err)
 	}
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusCreated {
 		var apiError *zoom.APIError
 
 		if err := json.Unmarshal(respBody, &apiError); err != nil {
-			return step.ExitCodeFailure, nil, fmt.Errorf("unmarshal response: %w", err)
+			return nil, status.FromError(codes.Internal, "Failed to unmarshal response", err)
```

Comment:
No need for "Failed", "Error", etc in error messages
```suggestion
			return nil, status.FromError(codes.Internal, "unmarshal response", err)
```